portfolio/src/main/webapp/scripts/comments.js

```diff
@@ -8,14 +8,34 @@ async function fetchAndInsertDataIntoDOM() {
 	commentsListElement.innerHTML = '';
 	comments.forEach(comment => {
 		commentsListElement.appendChild(
-			createListElement(comment)
+			createCommentElement(comment)
 		);
 	});
 }
 
-/** Creates an <li> element containing text */
-function createListElement(text) {
-	const liElement = document.createElement('li');
-	liElement.innerHTML = text;
-	return liElement;
+/** Creates a comment element containing */
```

Comment:
Im on the edge of my seat, what does it contain!?

Reply:
Hahaha, find out in the next horror filled episode of, "Interns Making Pull Requests".